webapps/ui/cockpit/plugins/base/app/views/common/diagramPlugins/styles.less

```diff
@@ -1,5 +1,5 @@
 .instances-overlay {
-  min-width: 100px;
```

Comment:
This used to create an invisible box which would mess with the hiding of the link overlay.